website/content/en/docs/ansible/reference/advanced_options.md

```diff
@@ -209,3 +209,56 @@ spec:
     served: true
     storage: true
 ```
+
+## Passing Arbitrary Arguments to Ansible
+
+Users can specify an arbitrary command-line argument to the ```operator-sdk exec-entrypoint``` and ```run --local``` subcommands for Ansible-based Operators via the ```ansible-args``` flag.
+
+**Example**
+
+The example below has ```--tags``` as the arbitrary command-line argument. Tags allow a playbook to run a specific part of the configuration without running the whole playbook. The aribtrary argument is specified as: 
+```
+operator-sdk run local --ansible-args='--tags "configuration,packages"'
+```
+```
+operator-sdk run local --ansible-args='--skip-tags "notification"'
+```
+Ansible-runner will perform the task relevant to the command specified by the user in the ```---ansible-args``` flag.
+
+
+## Support for Ansible-Vault
+
+The ```ansible-args``` flag also support the use of ```ansible-vault``` for file password storage, encryption, decryption, etc. Users can specify the password file as an aribitrary argument under the ```--ansible-args``` flag and have the operator perform the encryption/decryption. 
```

Comment:
```suggestion
[Ansible Vault][ansible-vault-doc] allows you to keep sensitive data such as passwords or keys in encrypted files, rather than as plaintext in playbooks or roles. You can specify Ansible-Vault file via an arbitrary argument by using the flag `--ansible-args`: 
```